File: algorithm-programmers-dfsbfs-target-number/src/test/java/com/currenjin/SolutionTest.java
package com.currenjin;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class SolutionTest {
    @Test
    void sample1() {
        assertEquals(5, new Solution().solution(new int[]{1, 1, 1, 1, 1}, 3));
    }

    @Test
    void sample2() {
        assertEquals(2, new Solution().solution(new int[]{4, 1, 2, 1}, 4));
    }
}

File: algorithm-programmers-dfsbfs-target-number/src/test/kotlin/com/currenjin/SolutionTest.kt
package com.currenjin

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class SolutionTest {
    @Test
    fun sample1() {
        assertEquals(5, Solution().solution(intArrayOf(1, 1, 1, 1, 1), 3))
    }

    @Test
    fun sample2() {
        assertEquals(2, Solution().solution(intArrayOf(4, 1, 2, 1), 4))
    }
}
